src/providers/LyricSify.ts

```diff
@@ -0,0 +1,71 @@
+import { fetchText } from '../utils'
+import { Provider, SearchParams } from './Provider'
+
+const BASE_URL = 'https://www.lyricsify.com/'
+
+export class LyricSify implements Provider {
+    private async getLink(artist: string, name: string) {
+        const query = artist + ' ' + name;
+        const data = await fetchText(BASE_URL + 'search?q=' + query);
+        const result = data.toString();
+        const list = result.split('<div class="li">');
+        const parsed = list.slice(1, list.length);
+        var links;
+        parsed.forEach((items) => {
+            const title = items.match('>(.*?)<')
+            const link = items.match('href=\"([^"]*)\"')
+            if (title?.length && link?.length) {
+                if (title[1].toLowerCase().includes(artist) && title[1].toLowerCase().includes(name)) {
+                    links = link[1];
+                }
+            }
+        });
+
+        return links;
+    }
+
+    private removeTags = (str: string) => {
+        if ((str === null) || (str === ''))
+            return false;
```

Comment:
```suggestion
        if (!str)
            return '';
```

Returning an empty string in this case seems to be more reasonable than returning a false.